### ISSUE TITLE
[BUG] SteamVR won't launch, preparing to launch SteamVR

### ISSUE THREAD

**Issue by DreamingRaven** [ValveSoftware/SteamVR-for-Linux#256]
**Describe the bug**
Attempting to launch SteamVR on up to date Archlinux install stops (aborts) after "preparing to launch SteamVR".

**To Reproduce**
1. Opt into steam client beta
2. Opt into SteamVR beta
3. Launch SteamVR
4. "Preparing to launch SteamVR"
5. nothing happens, popup closes

**Expected behavior**
Launches SteamVR into SteamVR home or at-least an appropriate error message is provided to be able to solve the issue.

**System Information (please complete the following information):**
- Distribution: Arch (```Linux azuran 5.3.1-arch1-1-ARCH #1 SMP PREEMPT Sat Sep 21 11:33:49 UTC 2019 x86_64 GNU/Linux```)
- SteamVR version: beta 1.8.6
- Steam client version: beta (unsure of how you find out)
- Opted into Steam client beta?: Yes
- Graphics driver version:  435.21
- Gist for SteamVR System Information: https://gist.github.com/DreamingRaven/f476c288f39057e1265ec0a97fc4d5e7 (cant launch SteamVR to follow the normal instructions)

**Screenshots**

**Additional context**
tried https://github.com/ValveSoftware/SteamVR-for-Linux/issues/160#issuecomment-444743731 which launches SteamVR fine (although valve index is not detected) It may be worth noting tho that my vrmonitor was not in the location he explained but instead in a subdirectory of ```SteamVR/bin/win32/vrmonitor.exe``` and in ```SteamVR/bin/win64/vrmonitor.exe``` instead of ```SteamVR/bin/vrmonitor.exe``` I don't know if this is related?

SteamVR integrity checks pass successfully.
Attempted system update still does not fix the issue.
System used to work until a system-wide update.
Works fine on a different partition on the same machine with an older archlinux install, but both used to work, so I know its not hardware.

**Note:** Commenters who are also experiencing this issue are encouraged to include the "System Information" section in their replies.

**Comment by Zamundaaa** [comment]
You can find the steam client version under help -> about steam.
Have you tried to launch Steam from the command line? Perhaps it shows some stuff there.
So you have .exe files in your SteamVR directory? And win* folders? That's not the case for me, and shouldn't be the case if you've installed SteamVR for Linux.
The binaries for vrmonitor and so on are in SteamVR/bin/linux64/ but the script to start it is in SteamVR/bin/
If you indeed  have .exe files then I'd guess you probably somehow enabled Steam to download the Windows version of SteamVR and subsequently that's what it tries to start or something? Double check if in the properties of SteamVR "Force the use of a specific Steam Play compatibility tool" is disabled.
Have you tried simply uninstalling and then installing again?

**Comment by DreamingRaven** [comment]
Hey so I don't know how I managed to get the Windows binaries installed automagically when they shouldn't have been, but after a full system reinstall everything worked fine. I wish I could have solved it with my existing install but reinstalling, deleting all the files, etc wasn't working, so I gave in to the dark side and orbital nuked it. Apologies for not being able to solve this issue for others in the future. All the best.
